DataFormats/L1TrackTrigger/interface/TTStub.h

```diff
@@ -28,34 +28,80 @@ class TTStub {
   /// Destructor
   ~TTStub();
 
-  /// Data members:   getABC( ... )
+  /// Data members:   ABC( ... )
```

Comment:
this contradicts the CMSSW [coding rules](https://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1) about naming of methods:

> ### 2 - Naming Rules
> 8\. Start method names with lowercase, use upper case initials for following words, e.g. `collisionPoint()`.
> ...
> 11\. For a getter method, using the value name is preferred, e.g. `momentum()`.

Reply:
fixed

Reply:
thanks... could you fix the comment as well ?

Reply:
fixed